usagecharge_test.go

```diff
@@ -30,9 +28,31 @@ func usageChargeTests(t *testing.T, usageCharge UsageCharge) {
 		RiskLevel:        &riskLevel,
 	}
 
-	if !reflect.DeepEqual(usageCharge, expected) {
-		t.Errorf("UsageCharge.Create returned %+v, expected %+v", usageCharge, expected)
+	if usageCharge.ID != expected.ID {
```

Comment:
If this is currently broken and prevents tests from passing even though they should pass then consider breaking them out into a separate PR to unblock that.

Reply:
ya it is currently broken for other builds.. good idea to move to another and easy enough

Reply:
https://github.com/bold-commerce/go-shopify/pull/97